app/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,37 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
+    <style name="AppTheme" parent="@style/Theme.AppCompat.Light.NoActionBar">
+        <item name="android:windowActionBar">false</item>
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorControlHighlight">@color/colorTranslucent</item>
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="Title" parent="android:Widget.TextView">
+        <item name="android:textSize">@dimen/text_title_size</item>
+        <item name="android:textColor">@color/blue</item>
+        <item name="android:singleLine">false</item>
+    </style>
+
+    <style name="Details" parent="Title">
+        <item name="android:textSize">@dimen/text_title_size</item>
+        <item name="android:textColor">@android:color/white</item>
+        <item name="android:layout_margin">@dimen/margin_small</item>
+    </style>
+
+    <style name="Attributes" parent="Title">
+        <item name="android:textSize">@dimen/text_size</item>
+        <item name="android:textColor">@android:color/darker_gray</item>
+        <item name="android:layout_margin">@dimen/margin_small</item>
+    </style>
+
+    <style name="NoResults" parent="android:Widget.TextView">
+        <item name="android:textSize">@dimen/text_title_size</item>
+        <item name="android:textColor">@color/blue_bright</item>
+        <item name="android:text">@string/list_empty</item>
+        <item name="android:background">@android:color/white</item>
+    </style>
+
```

Comment:
Many of the properties here seem like they are only used in one place, extracting these may be early optimisation. Styles are good for reusability... Setting `android:text` in a style is too much indirection, and unforeseen effects under the hood.

Reply:
it's a side project, so this has inherited many things I had before, for this doesn't really change a lot. but I see your point